addon/webextension/background/auth.js

```diff
@@ -157,6 +157,19 @@ window.auth = (function () {
     });
   };
 
+  exports.ensureLoggedIn = function () {
+    return browser.cookies.get({
```

Comment:
I don't think we need to worry about the cookies specifically, if we call `auth.authHeaders()` it should cover us.